drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -389,5 +389,11 @@ def auto_exit_configs
 
       Hmis::AutoExitConfig.all
     end
+
+    field :client_detail_forms, [Types::HmisSchema::OccurrencePointForm], null: false, description: 'Custom forms for collecting and/or displaying custom details for a Client (outside of the Client demographics form)'
+    def client_detail_forms
+      # No authorization required, this just resolving application configuration
+      Hmis::Form::Instance.active.with_role(:CLIENT_DETAIL)
```

Comment:
This prompted [this PT story](https://www.pivotaltracker.com/story/show/186821455), which I would like to implement very soon. Lets discuss there